test/tests/wasm/example.wast

```diff
@@ -0,0 +1,7 @@
+(module
+  (func $exampleImport (import "example" "exampleImport") (param i32) (result i32))
+  (func $exampleExport (export "exampleExport") (param $value i32) (result i32)
+    get_local $value
+    call $exampleImport
+  )
+)
```

Comment:
For future reader's reference, this is an "s-expression" textual representation, which was compiled to `example.wasm` using wabt's wast2wasm, from a build on master at [4492d0cbbb](https://github.com/WebAssembly/wabt/tree/4492d0cbbb0c008366dec0fd42ec242406ca01fe).

Textual representations of wasm are not yet spec'd so these s-expressions a convention--what little tooling exists so far mostly uses s-expressions. It's extremely likely that a totally different textual format will be standardized. It's mostly used because writing binary by hand is obviously even more impractical.